src/components/widgets/InfoCircleWithToolTip.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import {Box, makeStyles, Tooltip} from '@material-ui/core'
+import ItalicI from '../../assets/italic_i_icon.svg'
+import {latoFont} from '../../style/theme'
+
+const useStyles = makeStyles(theme => ({
```

Comment:
I think it makes sense here to take the color as a prop as opposed to hardcoding it to make it more reusable.